docs/resources/endpoint.md

```diff
@@ -1,65 +1,111 @@
 # Aptible Endpoint Resource
 
-This resource is used to create and manage endpoints for
+This resource is used to create and manage Endpoints for
 [Apps](https://www.aptible.com/documentation/deploy/reference/apps/endpoints.html)
 and
 [Databases](https://www.aptible.com/documentation/deploy/reference/databases/endpoints.html)
 running on Aptible.
 
 ## Example Usage
 
+### Simple Default Domain Endpoint
+
 ```hcl
-resource "aptible_endpoint" "example_endpoint" {
-    env_id = 123
-    process_type = "cmd"
-    resource_id = aptible_app.example_app.app_id
-    default_domain = true
-    endpoint_type = "https"
-    internal = false
-    platform = "alb"
-    container_port = 5000
-    managed = true
+resource "aptible_endpoint" "example" {
+  env_id         = data.aptible_environment.example.env_id
+  resource_id    = aptible_app.example.app_id
+  resource_type  = "app"
+  process_type   = "cmd"
+  default_domain = true
+}
+```
+
+## Managed Custom Domain
+
+This example creates a Managed TLS Endpoint with a custom domain and uses AWS
+Route53 to manage the Endpoint's DNS records:
+
+```hcl
+resource "aptible_endpoint" "example" {
+  env_id        = data.aptible_environment.example.env_id
+  resource_id   = aptible_app.example.app_id
+  resource_type = "app"
+  process_type  = "cmd"
+  managed       = true
+  domain        = "www.example.com"
+}
+
+data "aws_route53_zone" "example" {
+  name = "example.com."
```

Comment:
Don't need the `.` at the end "example.com"

Reply:
👍 I just copied from the AWS example.